cellmlmanip/model.py

```diff
@@ -458,12 +458,12 @@ def get_free_variable_symbol(self):
         # This should be unreachable
         raise ValueError('No free variable set in model.')  # pragma: no cover
 
-    def get_symbol_by_cmeta_id(self, cmeta_id):
-        """Searches the given graph and returns the symbol for the variable with the
-        given cmeta_id.
+    def _get_symbol_by_cmeta_id(self, cmeta_id):
```

Comment:
I'm not sure about this change. On the one hand it's internal at present. On the other hand other users of cellmlmanip might well find 'get symbol by id' functionality useful (e.g. because they're not as ontology focused as we are) and it doesn't need to be 'hidden'. So my inclination is actually to keep it public. The extra documentation should be sufficient to avoid user confusion.

Reply:
ok